providers/parameter.rb

```diff
@@ -51,6 +51,7 @@ def parameter_exists?(vhost, name)
 
     execute "set_parameter #{parameter}" do
       command cmd
+      environment 'HOME' => ENV.fetch('HOME', '/root')
```

Comment:
I think so. RabbitMQ or its CLI tools are typically do not run as `root`. Granted, `/var/lib/rabbitmq` isn't portable but it's the most sensible fallback value I can come up with.